examples/seismic/source.py

```diff
@@ -117,10 +117,6 @@ def __new__(cls, **kwargs):
 
         return obj
 
-    def __init__(self, *args, **kwargs):
-        if not self._cached():
-            super(PointSource, self).__init__(*args, **kwargs)
-
```

Comment:
Why?

Reply:
this is a no-op constructor basically, why should I leave it? it just calls the superclass constructor, which is gonna get called anyway w/o these lines